ci/cirrus-templates/script.bash

```diff
@@ -26,6 +26,7 @@ echo "Install Rustup using ./rustup-init.sh"
 
 sh rustup-init.sh --default-toolchain=stable --profile=minimal -y
 # It's the equivalent of `source`
+# shellcheck source=src/cli/self_update/env.sh
```

Comment:
Adding '-x' to the shellcheck invocation should fix this. See https://github.com/koalaman/shellcheck/issues/1012#issuecomment-359294974

Reply:
Thank you a lot! It works.